index.html

```diff
@@ -45,6 +45,174 @@ <h1>
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <ul class="tag_ctn">
+              <li class="tag"><span class="tag_txt">Ruby on Rails</span></li>
+              <li class="tag"><span class="tag_txt">CSS</span></li>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <ul class="tag_ctn">
+              <li class="tag"><span class="tag_txt">Ruby on Rails</span></li>
+              <li class="tag"><span class="tag_txt">CSS</span></li>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+    </section>
+  </section>
 
+  <!-- About Me Section -->
+  <section class="about_me">
+    <section class="head">
+      <div class="decor"> </div>
+      <h1>
+        About me
+      </h1>
+      <p class="description">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+      </p>
+      <section class="action_3 centered">
+        <button class="btn">
+          <span class="btn_txt">Get my resume</span>
+        </button>
+      </section>
+    </section>
+
+    <section class="about grid2-container">
+      <ul class="skills_list">
+      <section class="grid_2-item">
+        <section class="icon">
+            <section class="img">
+              <img src="images/Icon Langguage.png" alt="">
+            </section>
+        </section>
+        <section class="txt">
+        <li><p class="project_title">Languages</p></li>
+          <section class="tag_ctn">
+            <button class="tag"><span class="tag_txt">Javascript</span></button>
+            <button class="tag"><span class="tag_txt">Ruby</span></button>
+            <button class="tag"><span class="tag_txt">HTML</span></button>
+            <button class="tag"><span class="tag_txt">CSS</span></button>
```

Comment:
- [OPTIONAL] Following the UI, these are language lists and not buttons, so the best tag to use here should be a `ul` tag and each individual language should be a `li` tag. I'll suggest you implement this here and also all the other technology listed below as it follows the best semantics. :smile: :ringed_planet:

Reply:
Ok done